specification/maps/data-plane/Creator/preview/2022-09-01-preview/mapconfiguration.json

```diff
@@ -117,6 +117,13 @@
           {
             "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
+          {
+            "name": "Content-Type",
+            "description": "Specify the request body content type. `application/zip` if request is a zip file, `application/json` if request body is a JSON file.",
+            "type": "string",
+            "in": "header",
+            "required": true
+          },
```

Comment:
You should not specify the `Content-Type` header explicitly. Use `consumes`.

Reply:
Switched to use consumes